index.html

```diff
@@ -8,6 +8,14 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-HV2JL7ZN8B"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'G-HV2JL7ZN8B');
+    </script>
     <title>Portfolio</title>
```

Comment:
- [ ] Your report is looking good however after a deep search I have found something that you should also consider and include in your report.
### Page titles `<title>Portfolio</title>`.
It adequately and briefly describes the content of the page. For example:
#### Poor titles:
- Microverse project | Your portfolio.
- Portfolio project
#### Better page titles:
- Mark's portfolio
- Mark full-stack web developer

Reply:
You tagged the wrong Alex @andersonlebon, I'm the one you should have tagged. thanks for the review